R/write.R

```diff
@@ -22,7 +22,7 @@ write_union <- function(base_path, path, new_lines, quiet = FALSE) {
 }
 
 write_over <- function(base_path, path, contents) {
```

Comment:
Can you add a comment here, just to make clear that `content` should now be a vector?

Reply:
OK added this comment

``` r
## `contents` is a character vector of prospective lines
write_over <- function(base_path, path, contents) {
  ...
}
```

Reply:
Is this OK to merge? I'm going to try to bring the overall coverage up and it will be nice to have AppVeyor/Windows fully online.